config/config.go

```diff
@@ -111,11 +111,9 @@ func (l *Loader) Load(config interface{}) error {
 
 	l.v.AutomaticEnv()
 
-	var werr error
-
 	if err := l.v.ReadInConfig(); err != nil {
-		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			werr = ConfigFileNotFoundError{err}
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
```

Comment:
should we use `errors.As` instead?

```suggestion
		if errors.As(err, new(viper.ConfigFileNotFoundError) {
```